l10n_ch_payment_slip/invoice.py

```diff
@@ -102,26 +101,24 @@ def _compute_full_bvr_name(self, cr, uid, ids, field_names, arg,
                                context=None):
         res = {}
         move_line_obj = self.pool.get('account.move.line')
-        account_obj = self.pool.get('account.account')
-        tier_account_id = account_obj.search(
-            cr,
-            uid,
-            [('type', 'in', ['receivable', 'payable'])],
-            context=context
-        )
         for inv in self.browse(cr, uid, ids, context=context):
             move_lines = move_line_obj.search(
                 cr,
                 uid,
                 [('move_id', '=', inv.move_id.id),
-                 ('account_id', 'in', tier_account_id)],
+                 ('account_id', '=', inv.account_id.id)],
```

Comment:
are we sure that all move lines that need a bvr_ref always have the same account_id than the invoice?